### DISCUSSION TITLE
Unable to Install Copilot in Visual Studio 2022 - Missing IntelliCode

### DISCUSSION THREAD

**Discussion by pablo-meyer** [community/community#19310] [Copilot Conversations]
I'm getting the following error when attempting to install Copilot into VS 2022.

![image](https://user-images.githubusercontent.com/10983085/175393711-5edcc6a9-0072-45d9-ba41-416796954b1a.png)

I'm unable to click on its name, its not clickable.  I also attempted to manually install IntelliCode, but looks like I cant.

This is my VS version.

![image](https://user-images.githubusercontent.com/10983085/175393943-5acdf408-f71b-4ed4-9e63-7784be2f7129.png)

Microsoft Visual Studio Community 2022
Version 17.2.5
VisualStudio.17.Release/17.2.5+32616.157
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

ASP.NET and Web Tools 2019   17.2.393.26812
ASP.NET and Web Tools 2019

Azure App Service Tools v3.0.0   17.2.393.26812
Azure App Service Tools v3.0.0

C# Tools   4.2.0-4.22281.5+8d3180e5f00d42f0f0295165f756f368f0cbfa44
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

NuGet Package Manager   6.2.1
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

Razor (ASP.NET Core)   17.0.0.2218101+885a343b00bcab620a90c1550c37dafd730ce984
Provides languages services for ASP.NET Core Razor.

SQL Server Data Tools   17.0.62204.01010
Microsoft SQL Server Data Tools

TypeScript Tools   17.0.10418.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.2.0-4.22281.5+8d3180e5f00d42f0f0295165f756f368f0cbfa44
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   17.1.0-beta.22274.4+211fd760c2315d73cfa0d375fe64abb346bc1096
Microsoft Visual F# Tools

**Comment by belongtothenight** [comment]
I ran into the same issue as well, exactly the same alert.

**Comment by forgivesam168** [comment]
I ran into the same issue as well, exactly the same alert.

**Comment by Notloc** [comment]
Same problem :/

**Comment by Notloc** [comment]
I fixed the issue on my end.

- I opened Visual Studio Installer.
- Clicked 'Modify' on my Visual Studio 2022 installation
- Went to the 'Individual Components' tab
- Installed 'Intellicode'
- Installed Github Copilot as normal afterwards

**Reply by drusepth** [reply]
Thanks for these instructions. I had this error on 4 different computers today and was successfully able to install Copilot with these steps on all 4 of them. It's odd this is an undocumented prerequisite if it's not installed by default.
